src/System.Security.Cryptography.Cng/src/System/Security/Cryptography/CngKey.Export.cs

```diff
@@ -32,10 +32,11 @@ public byte[] Export(CngKeyBlobFormat format)
                 throw errorCode.ToCryptographicException();
 
             byte[] buffer = new byte[numBytesRequired];
-            errorCode = Interop.NCrypt.NCryptExportKey(_keyHandle, IntPtr.Zero, format.Format, IntPtr.Zero, buffer, numBytesRequired, out numBytesRequired, 0);
+            errorCode = Interop.NCrypt.NCryptExportKey(_keyHandle, IntPtr.Zero, format.Format, IntPtr.Zero, buffer, buffer.Length, out numBytesRequired, 0);
```

Comment:
The other call site passes a byte* rather than a byte[]. So I can’t compute the length of it. I could pass down the Length of the original array, but this doesn’t seem any more robust than what’s there now. Either way, the callsite isn’t self-correcting in the light of surrounding changes.